### PR TITLE
 fix: auto installation for centos 

### PR DESCRIPTION
**Pull request by liuhatry**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -72,5 +73,5 @@ export PATH="$HOME/.cargo/bin:$PATH"
 
 # install bagua
 #python3 -m pip install bagua -f https://repo.arrayfire.com/python/wheels/3.8.0/
-python3 -m pip install bagua-core==0.2.2.dev15 -f https://repo.arrayfire.com/python/wheels/3.8.0/ -i https://pypi.org/simple
+python3 -m pip install bagua-core==0.2.2.dev18 -f https://repo.arrayfire.com/python/wheels/3.8.0/ -i https://pypi.org/simple
```
Comment (NOBLES5E) [inline install.sh:76]: better with stable versions?

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: install.sh (hunk 1 above)
Comment (liuhatry) [reply install.sh]: done

File: install.sh [hunk 2]
```diff
@@ -46,11 +46,12 @@ if [ $OS_NAME == "Ubuntu" ]; then
 
   # install zlib, ssl, openmpi
   apt-get install -y zlib1g-dev libssl-dev openmpi-bin openmpi-doc libopenmpi-dev
-elif [ $OS_NAME == "CentOS Linux" ]; then
+elif [ "$OS_NAME" == "CentOS Linux" ]; then
   if [ $VERSION_ID == "7" ]; then
     yum install centos-release-scl-rh -y && yum install devtoolset-8-toolchain -y
-    scl enable devtoolset-8 bash
-    yum install http://repo.okay.com.mx/centos/7/x86_64/release/okay-release-1-1.noarch.rpm -y
+    #scl enable devtoolset-8 bash
```
Comment (NOBLES5E) [inline install.sh:52]: remove this line if not used

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**